src/CommonITILObject.php

```diff
@@ -5210,6 +5210,11 @@ public function updateDateMod($ID, $no_stat_computation = false, $users_id_lastu
                 $update['users_id_lastupdater'] = $users_id_lastupdater;
             }
 
+            // set take into account delay stat
+            if ($no_stat_computation && $this->getType() == Ticket::class) {
+                $update['takeintoaccount_delay_stat'] = 0;
```

Comment:
IMHO, we should not force the value here. `takeintoaccount_delay_stat` may have already been defined by a previous action.

Reply:
Like that it's okay ? 
```
if ($this->fields['takeintoaccount_delay_stat'] == 0) {
    $update['takeintoaccount_delay_stat'] = 0;
}
```